app/src/main/java/org/sopt/and/component/IconWithTitle.kt

```diff
@@ -0,0 +1,45 @@
+package org.sopt.and.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun IconWithTitle(
+    labelText: String,
+    actionText: String,
+    actionIcon: Painter,
+    onClick: ()->Unit
```

Comment:
```suggestion
    onClick: () -> Unit
```

Command + Option + L (Ctrl + Alt + L) 누르시면 자동정렬 됩니다!

Reply:
앗 자동정렬을 습관화해보겠습니다 ..!